apps/condo/migrations/20240925112456-0426_billingreceipt_billingreceipt_unique_context_and_importid.js

```diff
@@ -0,0 +1,50 @@
+// auto generated by kmigrator
+// KMIGRATOR:0426_billingreceipt_billingreceipt_unique_context_and_importid:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDQuMi4xMyBvbiAyMDI0LTA5LTI1IDA4OjI0Cgpmcm9tIGRqYW5nby5kYiBpbXBvcnQgbWlncmF0aW9ucywgbW9kZWxzCgoKY2xhc3MgTWlncmF0aW9uKG1pZ3JhdGlvbnMuTWlncmF0aW9uKToKCiAgICBkZXBlbmRlbmNpZXMgPSBbCiAgICAgICAgKCdfZGphbmdvX3NjaGVtYScsICcwNDI1X21lc3NhZ2VfbWVzc2FnZV91c2VyX3N0YXR1c19zZW50YXQnKSwKICAgIF0KCiAgICBvcGVyYXRpb25zID0gWwogICAgICAgIG1pZ3JhdGlvbnMuQWRkQ29uc3RyYWludCgKICAgICAgICAgICAgbW9kZWxfbmFtZT0nYmlsbGluZ3JlY2VpcHQnLAogICAgICAgICAgICBjb25zdHJhaW50PW1vZGVscy5VbmlxdWVDb25zdHJhaW50KGNvbmRpdGlvbj1tb2RlbHMuUSgoJ2RlbGV0ZWRBdF9faXNudWxsJywgVHJ1ZSkpLCBmaWVsZHM9KCdjb250ZXh0JywgJ2ltcG9ydElkJyksIG5hbWU9J2JpbGxpbmdSZWNlaXB0X3VuaXF1ZV9jb250ZXh0X2FuZF9pbXBvcnRJZCcpLAogICAgICAgICksCiAgICBdCg==
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+    --
+    -- [CUSTOM] Set Statement Timeout to some large amount - 25 min (25 * 60 => 1500 sec)
+    --
+    SET statement_timeout = '1500s';
+    --
+    -- Delete old (context, importId) constraint
+    --
+    ALTER TABLE "BillingReceipt" DROP CONSTRAINT "BillingReceipt_importId_9da6acbf_uniq";
+    --
+    -- Create constraint billingReceipt_unique_context_and_importId on model billingreceipt
+    --
+    CREATE UNIQUE INDEX IF NOT EXISTS "billingReceipt_unique_context_and_importId" ON "BillingReceipt" ("context", "importId") WHERE "deletedAt" IS NULL;
+    --
+    -- [CUSTOM] Revert Statement Timeout to default amount - 10 secs
+    --
+    SET statement_timeout = '10s';
+    COMMIT;
+
+    `)
+}
+
+exports.down = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+    --
+    -- [CUSTOM] Set Statement Timeout to some large amount - 25 min (25 * 60 => 1500 sec)
+    --
+    SET statement_timeout = '1500s';
+    --
+    -- Delete old (context, importId) constraint
+    --
+    ALTER TABLE "BillingReceipt" ADD CONSTRAINT "BillingReceipt_importId_9da6acbf_uniq" UNIQUE ("context", "importId");
```

Comment:
IF NOT EXISTS

Reply:
there's no such syntax for constraints, only IF EXISTS